images/miq-app/Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM centos:7
+FROM carbonin/manageiq-ruby-container
```

Comment:
We didn't go with SCL upstream because we didn't want to be held back waiting for newer rubies to be built.  We're close to switching to ruby 2.4 upstream, but there is no software collection for it yet.  Do you know when those packages will be available?